### PR TITLE
Form validation implemented according to specification

### PR DESCRIPTION
**Pull request by kkolade**
## The form validation is implemented: The content of the email field has to be in lowercase.
- When a user submits the form, check if the email field value is in lowercase
- If the validation is OK, the form is sent.
- If the validation is not OK,  an error message is shown to the user near the submit button informing them of the error, and the form is not sent..

### CODE REVIEW

**Comment by kkolade** [comment]
## PR observation resolved
- Linters files fixed
- README.md updated

**Review by Kossi-stack — CHANGES_REQUESTED** [review]
## Hi @kkolade,

**Good job so far! 👍🏻**
There are some issues that you still need to work on to go to the next project but you are almost there!💪

68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f556461535559436c723932636a57706556782f67697068792e676966

## To Highlight

✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ README file is descriptive.
✅ Commit messages are meaningful.

## Required Changes ♻️

- [ ] Kindly fix the error when i submit the form it does not go through.

https://www.loom.com/share/23415078e88746ec97a68e4427fc5850

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

_**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by kkolade** [comment]
## Fix PR comment: form not submitting
- The form now submits if no error

File: index.html [hunk 1]
```diff
@@ -215,6 +215,7 @@ <h2 class="contact__headline headline-text">Contact Me</h2>
             maxlength="500"
             required
           >Write your message here</textarea>
```
Comment (prolajumokeoni) [inline index.html:217]: ### OPTIONAL

- [ ] Kindly add `Write your message here` as a placeholder this way users don't need to clear text before typing the message

**Review by prolajumokeoni — APPROVED** [review]
Hi @Team

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
## Highlights
- No linter errors ✅ 
- Github flow was used correctly 🚀  

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- Check inline comment

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by kkolade** [comment]
## Textarea place holder issue now fixed
- The placeholder text in the textarea now clear off when user types in the textarea

**Comment by kkolade** [comment]
### Fixed textarea placeholder issue:
- Placeholder text in textarea now clears when the user starts to type
@prolajumokeoni

**Pull request merged**